deploy-review/solutions/microservice-session/src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+quarkus.devservices.enabled = false
+quarkus.openshift.route.expose = true
+quarkus.kubernetes.deploy = true
+quarkus.hibernate-orm.database.generation = drop-and-create
+
+quarkus.kubernetes-service-binding.services.expenses-db.api-version = postgres-operator.crunchydata.com/v1beta1
+quarkus.kubernetes-service-binding.services.expenses-db.kind = PostgresCluster
+quarkus.kubernetes-service-binding.services.expenses-db.name = pgcluster
+
+quarkus.rest-client."com.redhat.training.conference.speaker.SpeakerService".url=https://localhost:8082
+quarkus.rest-client."com.redhat.training.conference.speaker.SpeakerService".scope=javax.enterprise.context.ApplicationScoped
```

Comment:
The lab is for deploying apps in OCP, not sure if setting the REST client to talk to localhost will work on OCP.

Reply:
It's true and it will not be able to connect to localhost.